vm-gcp-typescript/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Looks like maybe this was generated with 4 spaces instead of 2?

Reply:
These generally _are_ four spaces. Only YAML is two-space, generally.